src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-group/info-group.component.html

```diff
@@ -1,6 +1,26 @@
 <div class="row">
-  <span class="info-group-title">{{ groupTitle }}</span>
+  <div class="info-group-title"
+       [ngbPopover]="popInfoTemplate"
+       #popInfo="ngbPopover"
+       triggers="">
+       {{ groupTitle }}
+    <button type="button"
+            class="popover-icon"
+            (click)="popInfo.toggle()">
+      <i [ngClass]="[icons.infoCircle]"></i>
```

Comment:
We should configure the size of the icon using its provided class
```suggestion
      <i [ngClass]="[icons.infoCircle, icons.large]"></i>
```